chanutils/concurrency_test.go

```diff
@@ -0,0 +1,57 @@
+package chanutils
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	errs = []error{errors.New("error #1"), errors.New("error #2")}
```

Comment:
nit: this could be defined in `TestErrGroup` so the var name is available in other contexts.

Reply:
fixed